lib/beetle/publisher.rb

```diff
@@ -2,6 +2,9 @@ module Beetle
   # Provides the publishing logic implementation.
   class Publisher < Base
 
+    DEFAULT_BUNNY_CONNECT_TIMEOUT = 5
```

Comment:
Isn't Bunny's `@disconnect_timeout` variable the one we really want to read?
There is no accessor for it yet, but I would vote for adding one via a monkey-patch.

Reply:
no patch required, as  `@disconnect_timeout = @read_write_timeout || @connect_timeout` and these can all be passed in when constructing the bunny instance as :socket_timout and :connect_timeout.

Reply:
I added the additional configuration option to beetle to pass down to bunny.